src/vdbeapi.c

```diff
@@ -1108,53 +1197,52 @@ static void columnMallocFailure(sqlite3_stmt *pStmt)
 ** The following routines are used to access elements of the current row
 ** in the result set.
 */
-
-void sqlite3_column_blob_(sqlite3_stmt *pStmt, int i) __EXPORT_NAME(sqlite3_column_blob) {
+void sqlite3_column_blob_(sqlite3_stmt *pStmt, int i) __attribute__((export_name("sqlite3_column_blob"))) {
```

Comment:
let's wrap this function with `#ifndef __sqlite_unmodified_upstream`

Reply:
Nice catch.